packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -23,6 +24,26 @@ const int defaultSplitterWidth = 10;
 const String runInProfileModeDocsUrl =
     'https://flutter.dev/docs/testing/ui-performance#run-in-profile-mode';
 
+// GA dimensions:
+String userAppType = ''; // dimension1
```

Comment:
are these intentionally set to the empty string?

Reply:
Done.